addons/website_sale/static/src/snippets/s_dynamic_snippet_products/000.scss

```diff
@@ -40,3 +40,9 @@
         transform: scale(.95);
     }
 }
+// Bootstrap's stretched links mess with the editor.
+// Since we just want to redirect on link click
+// we can safely ignore pointer events
+.s_dynamic_snippet_products .stretched-link::after {
+    pointer-events: inherit;
+}
```

Comment:
It works but I don't understand why :thinking: Is not inherit the same as auto (which is the default) ? :thinking: 

But actually, I don't think it would be the right way to fix the problem anyway:
- You add a visitor rule to solve an edit mode problem
- The template content is still editable, you should hide the problem by making click be simulated elsewhere somehow... (so you just avoid user to click inside the still-editable content). Or am I mistaken? The problem, should be fixed at the source: snippet options should not think they are to be instantiated on those non editable content

Reply:
@qsm-odoo Reviving this PR !

> It works but I don't understand why 

I don't understand why it worked. :/  I adapted the code to follow the rule that was first introduced by LTU (see commit message). It seems like that rule did not apply to pseudo elements so it now does.

> The template content is still editable, you should hide the problem by making click be simulated elsewhere somehow... 

Hopefully that's what pointer-event does now. It will ignore clicks on the pseudo elements and should only trigger a click on the section which is editable.

> snippet options should not think they are to be instantiated on those non editable content

I agree they should not, but at this time, SnippetsEditor can be started on o_not_editable and some snippets depend on it (in saas~15.2+ s_social_media is o_not_editable but you can click on icons to change them).
Maybe it should be adapted in master but it seemed to me that changing this behavior on a fix might be a bit much. What do you think ?